pkg/beacon/majority.go

```diff
@@ -498,3 +498,23 @@ func (m *Majority) UpstreamsStatus(ctx context.Context) (map[string]*UpstreamSta
 
 	return rsp, nil
 }
+
+func (m *Majority) GetCurrentSlot(ctx context.Context) (phase0.Slot, error) {
+	// Download the previous n epochs worth of epoch boundaries if they don't already exist
```

Comment:
I reckon we can actually just download these once and store `genesis`/`spec` on the `Majority` struct in some other loop, then this function just becomes that last return line

Reply:
updated, just ended up storing the `spec` on the `Majority` struct because the finalized Epoch